pkg/plugin/util/merge/merge.go

```diff
@@ -0,0 +1,150 @@
+package merge
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"path"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/dave/dst"
+	"github.com/dave/dst/decorator"
+	"golang.org/x/tools/go/ast/astutil"
+)
+
+func InsertCode(filename, code string) error {
```

Comment:
Let's first check the approach. I do not want you to spend an effort that will not be constructive.
However, as you know, in the plugin/utils, we have a helper to insert code, replace, etc.
So, we would need to use what we have already.

Reply:
Good point
I saw this code, but looks like it's just manipulate code as text, but this is manipulation code as code. 
But I understand, its enough for most cases, probably except the current one or similar few cases.